src/lib/falcor/FalcorRouter.js

```diff
@@ -1,4 +1,21 @@
 import BaseRouter from "falcor-router"
+import { ghostArticleQuery } from 'lib/ghostAPI'
+import { SQLArticleQuery, SQLAuthorQuery } from 'lib/SQL'
+import falcor from 'falcor'
+
+var $ref = falcor.Model.ref;  
+
+// Transform ghost API names into our names
+var ghostify = (name) => {
```

Comment:
Yeah I guess we can use the same names, I just did this instead as that's what we set up with Zane's initial posts.json, and then I thought that it maybe made sense to be able to do this at least for something as long as `published_at`. But yeah if Zane agrees we can just say that we use the same names as Ghost. And otherwise I'll change the function name, sure.